### ISSUE TITLE
Error about shadowing an artifact on Maven central

### ISSUE THREAD

**Issue by mikera** [clojars/clojars-web#639]
Hi I recently got the following error when attempting to deploy `net.mikera/clojure-pom` using Maven release plugin:

```
    [ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.7:deploy (default-deploy) on project clojure-pom: Failed to deploy metadata: Could not transfer metadata net.mikera:clojure-pom/maven-metadata.xml from/to clojars.org (https://clojars.org/repo): Access denied to: https://clojars.org/repo/net/mikera/clojure-pom/maven-metadata.xml, ReasonPhrase: Forbidden - shadowing Maven Central artifacts is not allowed (see https://git.io/vMUHN). -> [Help 1]
```

The reason appears to be a very old artifact on Maven central. I've been deploying to Clojars for several years since then.

Is there any way to continue deploying to Clojars?

**Comment by tobias** [comment]
When we implemented the shadow prevention feature, we discussed adding a whitelist for cases just like this. We opted to delay that implementation until someone asked for it, and here we are. I'll take a look at implementing this soon.

**Comment by tobias** [comment]
I've got a fix for this, but want to see if I can get in a fix for #640 as well before I deploy.

**Comment by mikera** [comment]
OK thanks. This one not particularly urgent.
